react/README.md

```diff
@@ -48,3 +48,10 @@ For example, to add XYZ to the `package.json` file in the `react` folder while d
 (This advice has come from https://github.com/BretFisher/node-docker-good-defaults)
 
 ## Testing
+Testing is done with React Testing Library and Jest.
+
+Test files are located in the same directory as the files they are testing. For example, CreateBox.test.js and CreateBox.tsx are both located in react\src\views\CreateBox. 
+
+For integration tests, we mock the Apollo client with a `MockedProvider` component instead of the `ApolloProvider` component that is used to handle real data. More information on mocking the Apollo client can be found at this link: https://www.apollographql.com/docs/react/development-testing/testing/
```

Comment:
Can you 'hide' the link URL behind e.g. 'this'? In Markdown it works like so: `[this](https://www.apollographql.com/docs/react/development-testing/testing/)`